src/Microsoft.Azure.WebJobs.Host/Blobs/Listeners/SharedBlobListener.cs

```diff
@@ -91,14 +91,8 @@ public void Dispose()
 
         private static IBlobListenerStrategy CreateStrategy(IStorageAccount account)
         {
-            if (!StorageClient.IsDevelopmentStorageAccount(account))
-            {
-                return new PollLogsStrategy();
-            }
-            else
-            {
-                return new ScanContainersStrategy();
-            }
```

Comment:
only tests are left referencing it, I should add a unit test for the new strategy too, but no other references